enzy_htp/analysis/spi.py

```diff
@@ -17,12 +17,40 @@
 
 from typing import List, Union, Any
 
-from enzy_htp.structure import (Structure, StructureEnsemble, Ligand, PDBParser, Residue)
+from enzy_htp.structure import (Structure, StructureEnsemble, Ligand, Residue)
 
 from plum import dispatch
-from enzy_htp._interface.pymol_interface import OpenPyMolSession
 
 
+@dispatch
+def spi_metric(
+    stru: StructureEnsemble,
+    ligand_sele: str,
+    pocket_sele: str 
```

Comment:
Change sele to mask or pattern. Assume no pymol knowledge outside of PyMol interface. (the pattern syntax is adapted as EnzyHTP selection pattern however sele is a term specific for pymol)